src/daft-recordbatch/src/ops/window.rs

```diff
@@ -89,8 +89,108 @@ impl RecordBatch {
         self.union(&window_result)
     }
 
+    #[allow(clippy::too_many_arguments)]
+    pub fn window_agg_dynamic_frame(
+        &self,
+        name: String,
+        agg_expr: &AggExpr,
+        min_periods: i64,
```

Comment:
Not a big issue but why is min_periods an i64? It should be unsigned right? I see that this is type used across the code base for min_periods.